docs/synapse/devguides/stormservices.rst

```diff
@@ -3,38 +3,114 @@
 Storm Service Development
 #########################
 
-NOTE: These docs are an initial place holder to hold some notes.
-
 Anatomy of a Storm Service
 ==========================
 
-Storm Service Modules
-=====================
+A Storm :ref:`gloss-service` is a standalone application that extends the capabilities of the Cortex.
+One common use case for creating a service is to add a Storm command that will query third-party data,
+translate the results into the Synapse datamodel, and then ingest them into the hypergraph.
+
+A Storm service generally implements the following components:
+
+- A :ref:`gloss-package` that contains the new :ref:`stormservice-cmd` and optional new :ref:`stormservice-mod`.
+
+- A subclass of ``synapse.lib.CellApi`` which uses the ``synapse.lib.StormSvc`` mixin and contains the following information:
+
+    - The service name, version, packages, and events as defined in ``synapse.lib.StormSvc``.
+    - Custom methods which will be accessible as Telepath API endpoints, and therefore available for use within defined Storm commands.
+
+- A subclass of ``synapse.lib.Cell`` which includes additional configuration defintions (``confdefs``) that the Storm service requires. Typically the methods required to implement the service functionality are contained here, and are called within the Cell API.
+
+Service startup
+---------------
+
+Since Storm services are Cell implementations, there are helpers available to simplify startup configuration.
```

Comment:
This isn't necessarily true, but implementing a stormservice from a Cell gives a bunch of very useful built-in machinery.